### PR TITLE
Simplify review UI

### PR DESCRIPTION
**Pull request by delvh**
Instead of always displaying all available actions as buttons, merge them into a single dropdown menu, same as GitHub. That decreases visual overload and is more mobile-friendly, while not losing any functionality.

## Screenshots
<details><summary>Before</summary>

grafik
</details> 
<details><summary>After (unexpanded)</summary>

grafik
</details>
 
<details><summary>After (expanded)</summary>

grafik
</details>

### CODE REVIEW

File: templates/repo/diff/box.tmpl [hunk 1]
```diff
@@ -159,25 +159,30 @@
...
 								{{if $isReviewFile}}
 									<label data-link="{{$.Issue.Link}}/viewed-files" data-headcommit="{{$.AfterCommitID}}" class="viewed-file-form unselectable{{if $file.IsViewed}} viewed-file-checked-form{{end}}">
 										<input type="checkbox" name="{{$file.GetDiffFileName}}" autocomplete="off"{{if $file.IsViewed}} checked{{end}}> {{ctx.Locale.Tr "repo.pulls.has_viewed_file"}}
 									</label>
 								{{end}}
+								<div class="ui dropdown basic">
+									{{svg "octicon-kebab-horizontal" 18 "icon tw-mx-2"}}
+									<div class="ui menu">
+										{{if not (or $file.IsIncomplete $file.IsBin $file.IsSubmodule)}}
+											<button class="ui unescape-button item">{{ctx.Locale.Tr "repo.unescape_control_characters"}}</button>
```
Comment (wxiaoguang) [inline templates/repo/diff/box.tmpl]: There is no `ui item` in Fomantic UI.

`ui xxx` means it is a top-level component. Any element in a component shouldn't have `ui ` prefix.

Comment (delvh) [reply templates/repo/diff/box.tmpl]: Done

**Comment by delvh** [comment]
Sidenote: I didn't even know we had an `edit file` button already.
For some reason, it was never shown…
All I did was move the code and add the `item` class and surrounding menu.

**Comment by lunny** [comment]
> Sidenote: I didn't even know we had an `edit file` button already. For some reason, it was never shown… All I did was move the code and add the `item` class and surrounding menu.

It's a new feature of v1.22, I would like to have a `delete file` and `rename file` in v1.23

**Review by wxiaoguang — APPROVED** [review]

**Review by lunny — APPROVED** [review]

**Comment by wxiaoguang** [comment]
Need to fix some bugs:  #32631 

1.  The menu would go behind the sticky header
2. The menu doesn't get initialized when "load more files"

**Pull request merged**